ulib/LowStar.Monotonic.Buffer.fsti

```diff
@@ -1045,6 +1088,10 @@ let modifies_liveness_insensitive_mreference_weak
 : Lemma
   (requires (modifies l h h' /\ address_liveness_insensitive_locs `loc_includes` l /\ h `HS.contains` x))
   (ensures (h' `HS.contains` x))
+  [SMTPatOr [
+    [SMTPat (h `HS.contains` x); SMTPat (modifies l h h');];
```

Comment:
ditto

Reply:
not remarking further on this issue below

Reply:
Thanks, I will revisit all of them.